user_tools/src/spark_rapids_pytools/cloud_api/dataproc_job.py

```diff
@@ -27,6 +27,14 @@ class DataprocLocalRapidsJob(RapidsLocalJob):
     job_label = 'dataprocLocal'
 
 
+@dataclass
+class DataprocSubmitSparkRapidsJob(RapidsSubmitSparkJob):
+    """
+    Implementation of a RAPIDS job that runs local on a local machine.
```

Comment:
Nit: I think the job here runs on a cluster. Please let me know if my understanding is incorrect here.

Reply:
Thanks @nartal1 !
Fixed it!